oneview-ansible.md

```diff
@@ -262,14 +263,14 @@ Manage the Artifact Bundle resource.
  Provides an interface to manage the Artifact Bundle. Can create, update, remove, and download, upload, extract
 
 #### Requirements (on the host that executes the module)
-  * hpOneView >= 3.1.0
```

Comment:
There are 494 deletions ongoing in this file, I see lots of resources having their documentation modified. 
Not entirely sure why that is. Have you tried rebasing from master and running `build.sh` to see if this still happens?